docs/whats_new.md

```diff
@@ -1,7 +1,9 @@
 # What's New
 
-Follow [@ASFHyP3](https://twitter.com/ASFHyP3 "https://twitter.com/ASFHyP3" ){target=_blank} on Twitter 
+Follow @ASFHyP3 on [Bluesky](https://bsky.app/profile/asfhyp3.bsky.social "https://bsky.app/profile/asfhyp3.bsky.social" ){target=_blank} 
+and [Mastodon](https://mastodon.social/@ASFHyP3 "https://mastodon.social/@ASFHyP3" ){target=_blank}
 to keep up to date on all things HyP3!
 
+You can also check out
 <a class="twitter-timeline" href="https://twitter.com/ASFHyP3">Tweets by ASFHyP3</a>
 <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
```

Comment:
I think we can just replace this with a mastodon feed directly using <https://mastofeed.com/>, something like:

```html
<div class="mastodon-timeline mastodon-timeline-rendered" style="display: flex; max-width: 100%; margin-top: 0px; margin-bottom: 0px;">
    <iframe allowfullscreen="true" allowtransparency="true" title="Mastodon Timeline"
            sandbox="allow-top-navigation allow-scripts allow-popups allow-popups-to-escape-sandbox"
            src="https://mastofeed.com/apiv2/feed?userurl=https%3A%2F%2Fmastodon.social%2Fusers%2FASFHyP3&theme=dark&size=100&header=true&replies=false&boosts=false"
            style="position: static; visibility: visible; width: 100%; min-width: 300px; min-height: 400px; display: block; flex-grow: 1;"
    ></iframe>
</div>
```

Reply:
Or this one might look better:
https://gitlab.com/idotj/mastodon-embed-timeline